src/GafferImage/ImageReader.cpp

```diff
@@ -63,6 +63,7 @@ ImageReader::ImageReader( const std::string &name )
 	:	ImageNode( name )
 {
 	storeIndexOfNextChild( g_firstChildIndex );
+	OpenColorIO::ConstConfigRcPtr config = OpenColorIO::GetCurrentConfig();
```

Comment:
Any reason to declare this so far away from the point it's actually used? I think putting it directly in front of the associated setValue() call would be clearer...